tools/src/android-update-native-dependencies/prompts.ts

```diff
@@ -0,0 +1,158 @@
+import chalk from 'chalk';
+import inquirer from 'inquirer';
+import stripAnsi from 'strip-ansi';
+
+import logger from '../Logger';
+import {
+  AndroidProjectDependenciesUpdates,
+  AndroidProjectReport,
+  GradleDependency,
+  GradleDependencyUpdate,
+} from './types';
+import { addColorBasedOnSemverDiff, calculateSemverDiff, getChangelogLink } from './utils';
+
+function generateAndroidProjectsSelectionChoice({
+  projectName,
+  gradleReport: { outdated, exceeded, unresolved },
+}: AndroidProjectReport) {
+  const name = `${projectName}${
+    outdated.length > 0 ? ` ${chalk.yellow(`(${outdated.length} ⚠️ )`)}` : ''
+  }${
+    exceeded.length > 0 || unresolved.length > 0
+      ? ` ${chalk.red(`(${exceeded.length + unresolved.length} ❗️)`)}`
+      : ''
+  }`;
```

Comment:
This doesn't look well, could you maybe move these interpolated values to separate variables?